src/filedialog.cpp

```diff
@@ -247,6 +247,12 @@ FileDialog::FileDialog(QWidget* parent, FilePath path) :
 
 FileDialog::~FileDialog() {
     freeFolder();
+    if(proxyModel_) {
```

Comment:
delete nullptr is a noop by design. So there is no point to guard it by if...

Reply:
Correct. Copy-pasted from pcmanfm-qt → `tabpage.cpp`. There's no harm.